src/components/Picker/Picker.test.jsx

```diff
@@ -0,0 +1,11 @@
+import {screen, render} from '@testing-library/react';
+import Picker from './Picker';
+
+it('it should render Picker Component', ()=> {
```

Comment:
These tests look good. Start small and simple and as you get more comfortable with the react testing library, add more to them